array_api_tests/test_fft.py

```diff
@@ -57,6 +57,9 @@ def draw_s_axes_norm_kwargs(x: Array, data: st.DataObject, *, size_gt_1=False) -
         s_strat = st.none() | s_strat
     s = data.draw(s_strat, label="s")
 
+    if axes is None and s is not None:
```

Comment:
It's better to just tell hypothesis to skip this case, like

```
assume(axis is not None or s is not None)
```

Reply:
Nice, thanks! Updated to the suggestion.